rosys/driving/keyboard_control_.py

```diff
@@ -33,7 +33,7 @@ def handle_keys(self, e: KeyEventArguments) -> None:
             self.steerer.update(self.direction.y, self.direction.x)
 
         # update individual speed components via arrow keys
-        if e.modifiers.shift and e.key.is_cursorkey:
```

Comment:
Why removing the SHIFT modifier?

Reply:
This is from a merge from main. I made have made a mistake when merging?

Reply:
It's not on main. It may have been accidentally introduced by myself when testing a demo. But `e.modifiers.shift` is wrong. I'll fix that on main.